.github/ISSUE_TEMPLATE/design.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Design
+about: Track or request design work
+title: 'Design [...] so that [...].'
+labels: design
+assignees: '@gissoo'
+
+---
+
+_This issue tracks design work through the [CDH design review workflow](https://princeton-cdh.github.io/docs/workflows/#workflow-5)._
+
+**Figma links**
+[desktop]() / [tablet]() / [mobile]()
```

Comment:
Figma links aren't usually in the description, are they? Gissoo typically shares links in the comments as she has things ready to share and then shares again as she revises them.  (These strike me as more useful for design implementation tasks, where we need them more.)

If/where we keep the Figma links, I would prefer we label the section more semantically instead of naming the specific platform we're currently using. (But I'm not sure what that is in this case — mockups?)

Reply:
You're right; I was thinking of the issues where we do design _testing_ where I normally put figma links for myself (or the tester) in the description. Maybe we could establish a convention of putting them in the description (and editing it) though, so that the most updated version is always at the top? Curious to know what @gissoo thinks.

Also agree that we don't need to name the platform explicitly since it could change; "mockups" or "design links" sounds good to me.